Dealtime/distribution/pom.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<!--
```

Comment:
License header seems to be not correct.

Reply:
fix with - https://github.com/wso2/product-apim-training/pull/2